apps/i18n/netsim/en_ploc.json

```diff
@@ -0,0 +1,3 @@
+{
+  "yes": "Yes"
```

Comment:
AFAIK, the idea of pseudoloc is to decorate all localized strings while keeping them readable (e.g. "Yes" -> "_Yes_"), so you can quickly see any unlocalized strings while using the app. Seems like a tool that should be kept in place. @tanya will know more.